internal/leases/bmc.go

```diff
@@ -29,6 +29,7 @@ func (i *ReportItem) EnrichWithBMCDetails(ipmiPort int, ipmiUser, ipmiPassword s
 		}
 	} else {
 		i.Log.Warn("could not retrieve bmc details of device", "mac", i.Mac, "ip", i.Ip, "err", err)
+		return err
```

Comment:
So, what is the implication of this? When skipping the error the reporting just sent part of the data and now it does not return the machine, I assume?

Reply:
The the calling errgroup is now vanishing the error :-) so no machine should be skipped if i am right